lightly/api/api_workflow_client.py

```diff
@@ -67,6 +72,13 @@ def __init__(self, token: str, dataset_id: str = None, embedding_id: str = None)
         self.scores_api = ScoresApi(api_client=api_client)
         self.samples_api = SamplesApi(api_client=api_client)
 
+    def check_version_compatibility(self):
+        minimum_version = get_minimum_compatible_version()
+        if version_compare(__version__, minimum_version) < 0:
+            raise ValueError(f"Incompatible Version of lightly pip package. "
+                             f"Please upgrade to at least version{minimum_version} "
```

Comment:
Add a space between `version` and `{minimum_version}` or do it like this `v{minimum_version}`